binding/python/README.md

```diff
@@ -1,225 +1,21 @@
 
 
-# Requirements
+# Installation
 
 ## On linux
-Please followed the [README](../../README.md#build) and have build and install multiverso successfully.
+Please followed the [README](../../README.md#build) to build and install multiverso.
 
 ## On windows
-You need MSBuild.exe installed and your system can find it in the $PATH. Then you should run [build_dll.bat](../../src/build_dll.bat) to build the .dll file and install the .dll. Multiverso doesn't have auto-installer for windows now, you have to copy the .dll to either system $PATH or the multiverso package folder.
+You need MSBuild.exe installed and make sure your system can find it in the $PATH. Then you should run [build_dll.bat](../../src/build_dll.bat) to build the .dll file and install the .dll. Multiverso doesn't have auto-installer for windows now, you have to copy the .dll to either system $PATH or the multiverso package folder.
```

Comment:
Add quote for `$PATH` (like this). and I suggest to use lists instead of paragraph description, something like.
```
1. Have `MSBuild.exe` installed and make it available in the `$PATH`.
2. Run [build_dll.bat](../../src/build_dll.bat) to build the `.dll` Dynamic Link Library file.
3. Copy/Install the Dynamic Link Library file to the system `$PATH` or multiverso package folder.
```